llvm/lib/Transforms/Instrumentation/MemProfiler.cpp

```diff
@@ -677,24 +677,10 @@ static void readMemprof(Module &M, Function &F,
                         const TargetLibraryInfo &TLI) {
   auto &Ctx = M.getContext();
 
-  auto FuncName = getIRPGOFuncName(F);
+  auto FuncName = F.getName();
```

Comment:
Can you add a comment summarizing why we specifically don't want to use getIRPGOFuncName here?